iocore/net/quic/QUICConnectionTable.cc

```diff
@@ -36,7 +36,13 @@ QUICConnectionTable::insert(QUICConnectionId cid, QUICConnection *connection)
 void
 QUICConnectionTable::erase(QUICConnectionId cid, QUICConnection *connection)
 {
-  ink_assert(this->_connections.get(cid) == connection);
+  QUICConnection *con = this->_connections.get(cid);
+  if (con == nullptr) {
+    Debug("ctable", "[%" PRIx64 "] can't find the vc ", static_cast<uint64_t>(cid));
```

Comment:
I'm ok with "ctable" for now but please start with "quic_", or "v_quic_" if it's  too noisy.

Reply:
will do.